packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -619,7 +619,12 @@ export default defineComponent({
           }
         ]
           .filter((field) => {
-            const hasField = get(firstResource, field.prop || field.name)
+            let hasField
+            if (field.prop) {
+              hasField = get(firstResource, field.prop) !== undefined
+            } else {
+              hasField = Object.prototype.hasOwnProperty.call(firstResource, field.name)
+            }
```

Comment:
why is this necessary?

Reply:
unittests /.-

Reply:
try the way we did it before and ResourceTable.spec.ts throws random errors with long stacktraces 😠

Reply:
Uargh - I assume this is because of the mocks we're using :-(
Wasn't able to fix it properly in a reasonable amount of time either.. :-\